### PR TITLE
Update k8s-install-gke.rst

### PR DESCRIPTION
**Pull request by ackerman80**
Edited the command for kube-dns restart

Signed-off-by: Arvind Soni [arvindsoni@gmail.com](mailto:arvindsoni@gmail.com)

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7056)

### CODE REVIEW

**Comment by ackerman80** [comment]
test-docs-please

File: Documentation/gettingstarted/k8s-install-gke.rst [hunk 1]
```diff
@@ -60,7 +60,7 @@ plugin. Restart kube-dns to ensure it is managed by Cilium.
 
 .. code:: bash
 
-     kubectl -n kube-system delete pod -l k8s-app: kube-dns
+     kubectl -n kube-system delete pod -l k8s-app=kube-dns
```
Comment (ianvernon) [inline Documentation/gettingstarted/k8s-install-gke.rst:63]: @ackerman80 please sign off your commit

Comment (ianvernon) [reply Documentation/gettingstarted/k8s-install-gke.rst:63]: @ackerman80 the commit is still not signed-off: 
screen shot 2019-02-12 at 5 31 51 pm

**Review by tgraf — APPROVED** [review]
Change itself looks good

File: Documentation/gettingstarted/k8s-install-gke.rst (hunk 1 above)
Comment (ackerman80) [reply Documentation/gettingstarted/k8s-install-gke.rst:63]: signed

**Pull request merged**